README.md

```diff
@@ -78,8 +78,9 @@ Add your test to `diff.sh`:
   
 ## How to name your branch  
   
-1. Take the last part of the url of your Trello ticket (e.g. "8-setup-feathers-js")  
-2. Name the branch after the Trello id (e.g. "8-setup-feathers-js")  
+1. Take the Ticket Number from JIRA (ticketsystem.schul-cloud.org), e.g. SC-999  
+2. Name the branch beginning with Ticket Number , e.g. `SC-999-fantasy-problem`
+3. Add (WIP), if it is still in progress
```

Comment:
Dieser Punkt könnte missverständlich sein, _"(WIP)"_ sollte nicht in den Branch Namen eingefügt werden sondern an den Pull Request. Wobei ich da auch einfach für das Label "WIP" plädieren würde.